packages/v4/patternfly-docs/content/design-guidelines/components/wizard/wizard.md

```diff
@@ -122,6 +122,19 @@ Optional steps can be added to the wizard. Make steps optional when they do not
 * The review step is enabled once all required steps have been filled out.
 
 * Once required steps have been filled, a tertiary button would appear to the right of the next/back buttons. Once clicked, this button would jump users ahead to the final review step. 
+ 
+ ### Wizard with a panel
```

Comment:
It looks like there's an extra space before this heading - unsure if this is related to the error in the build process but it's the only thing I'm seeing at the moment